plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -605,13 +616,21 @@ def ipa_connect(self, context=None):
             ipaadmin_principal = self.params_get("ipaadmin_principal")
             ipaadmin_password = self.params_get("ipaadmin_password")
 
+            # Get set of parameters to override in api.bootstrap().
+            _inv_override = ["ipaapi_context"]
```

Comment:
I am not sure that we should allow to pass any parameter to api_connect. In my opinion it would be good to have a allowed list also.

Reply:
Having this open to any `ipaapi_` variable would allow modules to specify their own connection variables using a "standardized" variable name.

Neither function that will use these parameters put any limit on them, and if they are not known to the function, they are passed along, or simply ignored.